app-drawer/app-drawer.html

```diff
@@ -108,24 +126,41 @@
     }
 
     #scrim {
-      position: fixed;
-      top: -120px;
+      position: absolute;
+      top: 0;
+      bottom: 0;
       left: 0;
       right: 0;
-      bottom: -120px;
       background: var(--app-drawer-scrim-background, rgba(0, 0, 0, 0.5));
       opacity: 0;
-      transition: opacity 0.2s ease;
+      visibility: hidden;
```

Comment:
Yes - if not, the scrim flashes visible when toggling `app-drawer-layout` from split to narrow layout (I broke this in a previous commit). Example demo/demo6.html.